PurchasesCoreSwift/Attribution/TrackingManagerProxy.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  TrackingManagerProxy.swift
+//  PurchasesCoreSwift
+//
+//  Created by Juanpe Catalán on 14/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+// TODO (post-migration): Make all the things internal again.
+@objc public enum FakeTrackingManagerAuthorizationStatus: Int {
+
+    case notDetermined = 0
+    case restricted
+    case denied
+    case authorized
+
+}
+
+// We need this class to avoid Kid apps being rejected for getting idfa. It seems like App
```

Comment:
wondering whether we should be clear in these comments that we won't actually try to improperly access idfa 😅

Reply:
It's a good question 🤔 I think this comment aims to clarify why this class exists for future contributors. So, I'm not sure if we should include here that we won't actually try to improperly access idfa. I think this info should be included in the doc of the library, shouldn't it? 
Although I don't have a strong opinion on that, so if we decide to include it, that's fine by me 🙂

WDTY?

Reply:
Definitely for future contributors, and maybe folks who are just curious if we're doing anything with idfa? People who might be auditing, etc. I'll add a little message because I think it's nice to point out.

Reply:
Thanks @taquitos :)